projects/core/components/button/button.style.less

```diff
@@ -81,7 +79,7 @@
 
     &[data-size='xl'] {
         height: 3.75rem;
-        font-size: @font-size-large;
+        font-size: var(--tui-font-size-m);
```

Comment:
Why not just use existing font variables?

Reply:
fixed